db/schema.rb

```diff
@@ -688,6 +688,7 @@
   create_table "notifications", id: :serial, force: :cascade do |t|
     t.string "action"
     t.datetime "created_at", null: false
+    t.boolean "hidden", default: false
```

Comment:
I named this column **hidden** while other columns are named **any_comments_hidden**  since there are other notifications that are not comments too.